ext/mbstring/tests/mb_ereg1.phpt

```diff
@@ -19,15 +19,16 @@ foreach ($a as $args) {
 }
 ?>
 ===DONE===
---EXPECTF--	
```

Comment:
@cmb69 Ah yeah sorry, this is automatic editor behavior...
I usually stage things interactively, but I missed those.